File: ios/model/Content.swift
struct Content: Serializable {
    let type: ContentType
    let content: String
    static func toObject(map: [String: Any]) -> Content? {
        return map["type"] == nil || map["content"] == nil ? nil : Content(type: ContentType(rawValue: map["type"]! as! String)!, content: map["content"]! as! String)
    }
}

File: android/model/Content.kt
package com.tobiasheymann.ep.model

import java.io.Serializable

data class Content(
    val type: ContentType,
    val content: String
) : Serializable {
    companion object {
        fun toObject(map: Map<String, Any>): Content? {
            return if (
                map["type"] == null
                || map["content"] == null
            )
                null
            else
                Content(
                    ContentType(map["type"]!! as String)!!,
                    map["content"]!! as String
                )
        }
    }
}
